packages/editor/src/envelope/LoadingScreen/LoadingScreen.tsx

```diff
@@ -47,9 +47,11 @@ export function LoadingScreen(props: { loading: boolean }) {
     }
   }, [props.loading]);
 
+  const style = !props.style ? "kie-tools--loading-screen" : props.style + " kie-tools--loading-screen";
```

Comment:
```suggestion
  const style = (props.style ? `${props.style} ` : "") + "kie-tools--loading-screen";
```
Not sure if this is better readable, but it'd be nice to remove the duplication of the hardcoded string `"kie-tools--loading-screen"`.